test/e2e-go/cli/goal/expect/goalExpectCommon.exp

```diff
@@ -85,7 +89,7 @@ proc ::AlgorandGoal::StartNode { TEST_ALGO_DIR {SYSTEMD_MANAGED ""} } {
 # Stop the node
 proc ::AlgorandGoal::StopNode { TEST_ALGO_DIR {SYSTEMD_MANAGED ""} } {
     set ::GLOBAL_TEST_ALGO_DIR $TEST_ALGO_DIR
-    set timeout 15
+    set timeout 5
```

Comment:
Reducing the timeout here may result in sporadic test failures on slower platforms.

Reply:
I'm ok setting it to 15; but please make sure not to extend it beyond 30. If the timeout takes more than 30, it means that the node internally failed to stop and we were forcing the process to start. ( which indicates of some locking issue )